Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN npm install -g /build/saf.tgz
 # Useful for CI pipelines
 RUN apk add bash jq curl ca-certificates
 
+USER ${USER:-node}
+
```

Comment:
I can't leave a comment in the appropriate place cause github.  I'm really confused about what's going on with that share directory thing - we're creating it in previous stage but trying to use it here + what're we even using it for in this stage?

Reply:
I'm not quite sure what any of this does either...

Reply:
figured it out btw - we're not using it in the second one's build process, it's more or less happenstance that the directory we're binding to to get the code copied over is the same as the one we tell people to use to put in their files and whatnot they want to have converted